packages/datadog-plugin-iovalkey/test/naming.js

```diff
@@ -0,0 +1,19 @@
+const { resolveNaming } = require('../../dd-trace/test/plugins/helpers')
+
+const rawExpectedSchema = {
+  outbound: {
+    v0: {
+      opName: 'redis.command',
```

Comment:
I need to figure out if it's better to have a `valkey.command` option here. The two services are very similar but I'm not sure what the UI implications are. It seems that long term we'd be better off with `valkey.command` but in the short term using `redis.command` probably "just works".